currency_rate_update/README.rst

```diff
@@ -14,11 +14,14 @@ The reference rates are based on the regular daily query procedure between centr
 Configuration
 =============
 
-To configure the module, go to *Accounting > Configuration > Multi-currencies > Rate Auto-download* and create one or several services to download rates from the Internet.
+To configure the module:
 
-Then, go to the page *Accounting > Configuration > Settings* and, in the section *Multi Currencies*, make sure that the option *Automatic Currency Rates Download* is enabled.
+1. Go to *Invoicing > Configuration > Rate Auto-download* and create one or several services to download rates from the Internet.
+
+2. Go to the page *Invoicing > Configuration > Settings* and, in the section *Multi Currencies*, make sure that the option *Automatic Currency Rates Download* is enabled.
+
+3. In developper mode, in the menu *Settings > Technical > Scheduled Actions*, make sure that the action *Currency Rate Update* is active. If you want to run it immediately, use the button *Run Manually*.
 
-In developper mode, in the menu *Settings > Technical > Scheduled Actions*, make sure that the action *Currency Rate Update* is active. If you want to run it immediately, use the button *Run Manually*.
 
 This module is compatible with OCA module 'currency_rate_inverted' also found in OCA/currency repository, that allows to maintain exchange rates in inverted format, helping to resolve rounding issues.
```

Comment:
Actually the module currency_rate_inverted is not available for v11: I would remove the sentence